packages/privacy-toolset/src/cookie-banner/styles.scss

```diff
@@ -10,7 +10,7 @@ $horizontal-padding: 20px;
 $cookie-banner-bg-color: #fff;
 $cookie-banner-text-color: #000;
 $cookie-banner-text-muted-color: #000;
-$cookie-banner-link-color: #117ac9;
+$cookie-banner-link-color: #3858e9;
```

Comment:
Isn't there a variable we can reference for this? 🤔

Reply:
There is. I was just opening a separate issue for an alternate approach, which is:

* Update all hardcoded instances to refer to `----studio-blue-50`
* Merge a version of https://github.com/Automattic/color-studio/pull/764, which updates the values of Blue 50 to the blueberry.

What do you think?

Reply:
Oh yeah, that makes sense!